projects/schematics/src/migrations/mechanism/update-ssr/update-ssr-files.ts

```diff
@@ -4,8 +4,77 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { chain, Rule } from '@angular-devkit/schematics';
+import {
+  chain,
+  Rule,
+  SchematicContext,
+  Tree,
+} from '@angular-devkit/schematics';
+import { Path } from '@angular-devkit/core';
+import * as ts from 'typescript';
+import { EXPRESS_TOKENS, SSR_SETUP_IMPORT } from '../../../shared/constants';
 
 export function updateServerFiles(): Rule {
-  return chain([]);
+  return chain([removeExpressTokensFile, updateTokenImportPaths]);
+}
+
+function removeExpressTokensFile(): Rule {
+  return (tree: Tree, _context: SchematicContext) => {
+    tree.visit((filePath: Path) => {
+      if (filePath.endsWith(`${EXPRESS_TOKENS}.ts`)) {
+        tree.delete(filePath);
+      }
+    });
+    return tree;
+  };
+}
+
+function getExpressImportPaths(source: ts.SourceFile): string[] {
+  const importPaths: string[] = [];
+  source.statements.map((node) => {
```

Comment:
`query:` Do we need to use `map` here? Is mapping values inside source statements required or do you need to iterate over them?

Reply:
I've changed it to foreach